ignition/templates/driver_template/docs/index.md

```diff
@@ -35,8 +35,6 @@ Reference the values file on install to apply configuration:
 helm install --name {(helm.name)} {(helm.name)}-{(app.version)}.tgz -f custom_values.yaml-
 ```
 
-Once the pod for the driver has started you will be able to access the Swagger UI at: `http://<kubernetes-node-ip>:{(helm.node_port)}/api/driver/ui`
```

Comment:
Can we replace this with the following:

"Once the pod for the driver has started you should be able to view the status of the application at: ` `http://<kubernetes-node-ip>:{(helm.node_port)}/management/health`"

This ensures the user has a way of confirming the app is running.

Reply:
Added the following as suggested:

Once the pod for the driver has started you should be able to view the status of the application at: `http://:{(helm.node_port)}/management/health`

Reply:
@dvaccarosenna Please review again and let me know if any other change is needed.

Reply:
Looks like Git reformatted by suggestion and removed part of it. 

The suggested URL should be: `http://<kubernetes-node-ip>:{(helm.node_port)}/management/health`

Reply:
Updated the correct URL